document_clipper/pdf.py

```diff
@@ -150,14 +150,10 @@ def get_page_max_dimensions(self, page_tag_node):
         return self._str_list_to_int_list(params)
 
     def _convert_to_jpg(self, image_path):
-        img = Image.open(image_path)
-        if img.mode != "RGB":
-            img = img.convert("RGB")
-            base = os.path.splitext(image_path)[0]
-            new_image_path = base + '.jpg'
-            img.save(new_image_path, 'JPEG')
-            image_path = new_image_path
-        return image_path
```

Comment:
@jlmadurga For non-RGB images, there was a conversion step (`img.convert("RGB")`). I am not sure if omitting this step for all images will produce bad results.

Reply:
@nick13jaremek Yes there was, but formats like ppm (common in some PDFs) are RGB but not compatible with OCRs like google vision so it is needed to convert to jpg.
Converting all to jpg avoid format not accepted errors

Reply:
@jlmadurga Indeed, converting every image to JPG is a great solution to avoid format errors. But would you also convert the color space to RGB, to obtain RGB-only JPG images?

If the color space is not a problem for the resulting images, then the color space conversion is not needed, and this code is good as-is.

Reply:
@nick13jaremek pilkit `save_image` function calls `prepare_image` which is in charge of converting to RGB depending mode
https://github.com/matthewwithanm/pilkit/blob/cf60f3cbc28a15db46acfda1e61a704efad31c23/pilkit/utils.py#L272

Using jpg format may cause losing some quality but I chose to avoid format errors. In fact there are multiple discussion if converting to other modes as grey-scale will improve vision results